core/modules/config/tests/config.test

```diff
@@ -158,6 +158,21 @@ class ConfigurationTest extends BackdropWebTestCase {
     $this->assertEqual(FALSE, $config_data->isOverridden('foo.key1'), 'Original value no longer overridden.');
     $this->assertEqual($config_data->get('foo.key1'), 'chocolate', 'Original value has been updated to include overridden value.');
   }
+
+  /**
+   * Tests that config directories are protected by .htaccess files.
+   */
+  public function testConfigHtaccess() {
+    // Remove .htaccess file to then test that it gets re-created.
+    @backdrop_unlink(config_get_config_directory() . '/.htaccess');
+    $this->assertFalse(is_file(config_get_config_directory() . '/.htaccess'), "Successfully removed the .htaccess file in the 'active' config directory.");
+    file_save_htaccess(config_get_config_directory());
```

Comment:
Instead of calling `file_save_htaccess()`, could you do a normal web request and confirm that the .htaccess file is remade as needed?

Reply:
Not sure what you mean, sorry...

Reply:
Oh, you mean access the Status Report page to see if that triggers it?

Reply:
Yeah that would be perfect.

Reply:
resolved in new PR https://github.com/backdrop/backdrop/pull/3279

Reply:
Hm. the tests are still failing. I've confirmed the file is there on disk but these tests can't see it.